notebooks/json_blocks/WET/BLOCK-T419.ipynb

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            "description": "Day of the year for the reference state.",
</pre><p>See my comments in the WET-001 notebooks about <code>day</code> and <code>seq</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>